compiler/luci/profile/include/luci/Profile/CircleNodeMemoryPlan.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
not sure this file is profile related.
Need to find more appropriate place

Reply:
> Not sure this file is profile related.

Agree :)

Reply:
@seanshpark 
What do you think about separate directory, something like "ExecutionPlan"?

Different approach I see is:
- introduce "plugin" interfaces in luci importer and exporter to injecting external code
- place plan related code in luci-interpreter/somewhere else

Reply:
I think first one looks better for now. Second one may take some time but maybe better for future extensions...
Does 'MemoryPlan' need profile stuff? I didn't look in to the code so don't know this question is valid...
If `profile` and memory planning are not related then can we think of adding something like `luci/plan` ?
This `plan` or `planning` can provide memory and execution planning. 
Or any better word is welcome for purpose of execution optimization. (as I understand this to provide information for better execution performance)

Reply:
As I understand `profile` and memory plan are not related. And that is why, yes, i think adding `luci/plan` can be good idea. @binarman what do you think about it?

Reply:
> Does 'MemoryPlan' need profile stuff?

> As I understand profile and memory plan are not related

For now I don't see how these two could be related =)

I think "plan" is good =)

@seanshpark FYI
> as I understand this to provide information for better execution performance

This should reduce **initialization** time and **memory consumption** of luci-interpreter (maybe other runtimes, if they will to use this annotations)

Reply:
FYI when `luci_plan` gets to exist, we need to register to `infra/debian/compiler/one-compiler.links`

@mhs4670go , is there any other place to register new file?

Reply:
@seanshpark Just registering it to `infra/debian/compiler/one-compiler.links` is enough.